### DISCUSSION TITLE
Do cloud deployments support raster tiles?

### DISCUSSION THREAD

**Discussion by MzicoDim** [protomaps/PMTiles#190] [Q&A] [answered]
Following the docs for the cloud deployment at https://protomaps.com/docs/cdn
There is no mention of Raster or Vector tiles, but .mvt is the extension the serverless function returns. 
MVT stands for Map Vector Tiles https://github.com/mapbox/vector-tile-spec which wouldn't be compatible with raster content.

This must sound pretty obvious, but a mention of the limitation in the docs would be very useful.

**Comment by bdon** [comment, marked as answer]
The serverless function supports .pbf/.mvt, .jpg, and .png - it will return a 4xx response If the requested extension does not match the archive metadata. So it should work just fine with raster tiles (please report a bug if not)

**Comment by MzicoDim** [comment]
Sorry for the misunderstanding, would love if the docs mentioned that mvt is only one of multiple choices.

The error did mention a wrong type, but I assumed it meant only mvt/vector tiles were supported
![chrome_XW0BbELZwk](https://github.com/protomaps/PMTiles/assets/134812394/d2f4363b-5aaa-4ff3-afa0-2dd372858503)

**Reply by bdon** [reply]
Good catch, here's a PR to improve the error message: https://github.com/protomaps/PMTiles/pull/191 what do you think?

**Reply by MzicoDim** [reply]
PR looks good, I understand this is not the repo for the docs, but I would add a little hint there as well.

**Reply by bdon** [reply]
Thanks, I've updated the docs site: https://protomaps.com/docs/cdn

**Reply by MzicoDim** [reply]
👍 awesome
